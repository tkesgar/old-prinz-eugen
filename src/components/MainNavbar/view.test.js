import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import ViewComponent from './view'

describe('MainNavbar', () => {
  const wrapper = mount(ViewComponent, { localVue })

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
