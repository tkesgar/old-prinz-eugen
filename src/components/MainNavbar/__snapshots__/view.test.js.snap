// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNavbar should match snapshot 1`] = `
<nav
  class="navbar navbar-dark bg-dark navbar-expand"
>
  <a
    class="navbar-brand"
    href="/"
    target="_self"
  >
    
    Prinz Eugen 
    <sup>
      alpha
    </sup>
  </a>
   
  <!---->
   
  <div
    class="ml-auto"
  >
    <ul
      class="navbar-nav"
    >
      <li
        class="nav-item"
      >
        <a
          class="nav-link"
          href="/auth/google"
          target="_self"
        >
          Masuk
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
